src/components/Stats/Stats.js

```diff
@@ -21,11 +23,14 @@ const Stats = ({
       rootTagName="span"
       rootProps={{ className: cssClasses.text }}
       data={{
+        hasManySortedHits: nbSortedHits > 1,
         hasManyResults: nbHits > 1,
         hasNoResults: nbHits === 0,
         hasOneResult: nbHits === 1,
```

Comment:
we might need these conditions for smart sort too

Reply:
https://github.com/algolia/instantsearch.js/pull/4649/commits/9ad4b886c6933bafad751fb4efa9aae7d7ed8588